Dockerfile

```diff
@@ -12,15 +12,18 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM alpine:3.12
+FROM alpine:3.13
 
 LABEL com.github.actions.name="Nancy for GitHub Actions" \
     com.github.actions.description="Run Sonatype Nancy as part of your GitHub Actions workflow."
 
-RUN apk add --no-cache curl && \
-    curl -L -o nancy.apk \
-        https://github.com/sonatype-nexus-community/nancy/releases/download/v1.0.0/nancy_1.0.0_linux_386.apk && \
-    apk add --no-cache --allow-untrusted nancy.apk
+# required to fetch nancy.apk via curl
+RUN apk add --no-cache curl
+
+# required to get grep that supports -P option
+RUN apk add --no-cache --upgrade grep
```

Comment:
Not a major thing, but I think a good idea in general is to clean up after yourself, so maybe delete these packages unless they are needed?

Reply:
I think the upgraded `grep` must stick around because it is used later when running the docker image (to detect the "latest" version).

Reply:
Yeah that makes sense. I'd ditch curl maybe unless you need it? It's not a huge deal but I personally love leaving the environment as pristine as possible.